### PR TITLE
#21 fix .widget-container size issue

### PR DESCRIPTION
**Pull request by shye0000**
I tried to resolve the css bug I mentioned in this issue: [https://github.com/Wolox/react-chat-widget/issues/21](https://github.com/Wolox/react-chat-widget/issues/21)

### CODE REVIEW

**Comment by mcallegari10** [comment]
Hi! Thank you so much for contributing to this project! Now, I've tested what you change in this PR, but this renders something like this 

image

I've seen that the problem is the `height: 100%;` in the `widget-container`'s media query. It's no wrong but it shouldn't be there all the time, so the fix is to add it when the launcher is clicked :smile: 

image

**Comment by shye0000** [comment]
@mcallegari10 hi, I corrected it, could you please review it? thanks

**Comment by pauljmillar** [comment]
Hey There. Curious if this fix was ever reviewed?

**Comment by Thomas-Negrault** [comment]
I'm having the same problem, on a progressive web app, on a mobile (responsive), the UI is inaccessible even with the chat closed.

File: src/components/Widget/style.scss [hunk 1]
```diff
@@ -9,7 +9,6 @@
   max-width: 370px;
   position: fixed;
   right: 0;
-  width: 90vw;
   z-index: 9999;
```
Comment (Thomas-Negrault) [inline src/components/Widget/style.scss]: With your changes this is the result on my phone (iPhone 7)
img_2740

In order to fix this bug I added:
`max-height: 100%;`
In `.widget-container {...`. Here is the result:

img_2741

**Comment by tammaroivan** [comment]
Fixed in #51

**Pull request closed without merging**